python/lsst/summit/utils/utils.py

```diff
@@ -601,3 +606,46 @@ def getExpPositionOffset(exp1, exp2, useWcs=True, allowDifferentPlateScales=Fals
                           )
 
     return ret
+
+
+def starTrackerFileToExposure(filename, logger=None):
```

Comment:
This should probably be added to __all__

Reply:
Good shout, yes, thanks 🙂